servicex/servicex_client.py

```diff
@@ -183,7 +183,9 @@ def get_codegen(_sample: Sample, _general: General):
 
     sx = ServiceXClient(backend=servicex_name, config_path=config_path)
     datasets = []
+    title_length_limit = make_sync(sx.servicex.get_servicex_sample_title_limit)()
```

Comment:
Is this called inside a async? I'm worried that we move in and out of async here - the only reason this wasn't async was because we didn't use it that way. For example, a `async` version of `deliver`?

Reply:
Ok - this is called directly from `deliver`, and `deliver` for whatever reason has no async version. So... I guess this is needed.